tecken/apps.py

```diff
@@ -6,17 +6,43 @@
 
 import markus
 from django_redis import get_redis_connection
-import sentry_sdk
+from sentry_sdk.integrations.boto3 import Boto3Integration
 from sentry_sdk.integrations.django import DjangoIntegration
+from sentry_sdk.integrations.redis import RedisIntegration
 from sentry_sdk.integrations.logging import ignore_logger
 
 from django.conf import settings
 from django.apps import AppConfig
 
+from tecken.libdockerflow import get_release_name
+from tecken.libsentry import (
+    build_scrub_query_string,
+    scrub,
+    Scrubber,
+    SCRUB_KEYS_DEFAULT,
+    set_up_sentry,
+)
+
 
 logger = logging.getLogger("django")
 
 
+SCRUB_KEYS_TECKEN = [
+    # HTTP request bits
+    (
```

Comment:
I'm not thrilled about how this is defined. I tossed around some other possibilities that weren't parens heavy, but they seemed overly verbose.

The dataclass approach would be something like this:

```
SCRUB_KEYS_TECKEN = [
    ScrubRule(
        parent="request.headers",
        keys=("Auth-Token", "Cookie", "X-Forwarded-For", "X-Real-Ip"),
        actions=scrub
    )
    ...
]
```

That's much clearer. We really want clarity here, so maybe that's the right thing to do.